src/apps/search-result/search-result.entry.tsx

```diff
@@ -51,10 +50,13 @@ const SearchResultEntry: React.FC<SearchResultEntryProps> = ({
   // Get number of result items to be shown.
   // If the number of items has been defined with data attributes use those
   // otherwise get them from the configuration.
-  const pageSize = getPageSize({
-    desktop: pageSizeDesktop,
-    mobile: pageSizeMobile
-  });
+  const pageSize = getPageSize(
+    {
+      desktop: pageSizeDesktop,
+      mobile: pageSizeMobile
+    },
+    "pageSize"
```

Comment:
Could we maybe rename this scope to: "pageSizeGlobal" or "pageSizeDefault" or similar.
Or maybe even better make the parameter optional and default it to "pageSize".
Seems redundant to call a function called "getPageSize" and then give it a configuration scope name called: "pageSize". it's like _duh_! :)

Reply:
I will rename :)